### DISCUSSION TITLE
Population density dataset

### DISCUSSION THREAD

**Discussion by znichollscr** [PCMDI/input4MIPs_CVs#188] [Announcements]
Some groups need population information apparently. From the discussion [here](https://office.wcrp-cmip.org/Products/Projects/Messages.aspx?prjID=33&id=327#comments) (not public link, sorry, but all the info is now here):

- @ssmithClimate seemed to be our key contact for this. The last discussions were about contacting Bryan Jones (CUNY) and Detlef or someone about HYDE (Utrect U.). 
- Then there is also the note by @vnaik60 that  the [SSP population projections](https://iopscience.iop.org/article/10.1088/1748-9326/11/8/084003/meta) used [GPWv3](https://sedac.ciesin.columbia.edu/data/collection/gpw-v3/sets/browse), which may or may not be the same as HYDE.
- whatever dataset we end up with, we need to connect with the data provider and go through the usual process. Ideally we find the same data provider for historical and scenarios to make harmonisation easier.

**Comment by durack1** [comment]
WorldPop/FuturePop grant - supported by https://wellcome.org/

Contacts below:

```
From: Felipe Colon Gonzalez @wellcome.org
Sent: Tuesday, February 4, 2025 10:26 AM
To: Joy Shumake-Guillemot @wmo.int; Andrew Tatem @soton.ac.uk; Laurence Hawker @bristol.ac.uk
Cc: Donna James @wellcome.org
Subject: [Introduction] Introducing the FuturePop/WorldPop team

Dear Joy,

I hope this email finds you well. I would like to introduce you to the PIs of our WorldPop/FuturePop grant.
They've been awarded some funds to develop a new gridded dataset of global population projections for
all SSP scenarios at a high spatial resolution. Moreover, they will disaggregate those estimates by age
brackets and sex something that has not been attempted before. WorldPop has also been awarded some
funds to support the main data assets of spatiotemporal data they host and that are highly relevant to
the C&H community.

We believe that by putting your team and theirs in touch you could collaborate in meaningful ways to
strengthen the C&H field. We hope this connection leads to effective and long-standing collaborations.

I will leave it to Andy and Laurence to tell you more about the great work they do.

Best wishes,

Felipe

Dr. Felipe Colón (He/Him/His)
(/fɛˈli pɛ/, [feˈli.pe] /, phe lee peh)
Technology Lead
Data for Science and Health
Wellcome Trust
Gibbs Building
215 Euston Road, NW1 2BE, UK
E: [firstInitial.lastName@wellcome.org]
```

**Comment by eleanororourke** [comment]
I am in contact with the Future Pop/World Pop team and they are keen to meet, which I will set up this week (with a Doodle). Some additional info from them:

Our original project timeline had a target of Q2 2026 for a release in Africa which would then be rolled out to the global by the end of 2026.

However, an 'alpha' version of FuturePop population counts already exists for the globe and I would expect a good 'beta' standard product to be available by the end of this year. Age/sex disaggregated maps would lag around 6 months behind these.

It would be great to have a call to find out requirements. For instance if a particular region is of most interest we can focus on that (but I assume you need global).

 We are very keen to be involved and hope we can come to some sort of agreement to provide you with what you need.

**Comment by vnaik60** [comment]
On historical population density dataset, we have been approached by Fang Li for the possiblity of sharing the HYDE3.4 dataset that she processed for the CESM model. 


```
Dear Paul and Vaishali,

I am writing as the fire model developer of CESM and co-chair of CMIP7 FireMIP.
I noticed that the CMIP7 population density forcing is currently marked as TBD. As population density is a crucial input for ESM fire modeling, I would like to share a relevant dataset I recently developed.
This is a global 0.5-degree annual population density dataset spanning 1850-2023, derived from HYDE3.4 population data (decadal before 1950 and annual for 1950-2023). The dataset will serve as forcing data for CESM3 (https://github.com/ESCOMP/CTSM/issues/2701#issuecomment-2635071041). 

I would be happy to provide this dataset if it would be useful for CMIP7.

Best regards,
Fang

________________________________________
Fang Li (she, her, hers) 
Professor (Full), 
International Center for Climate and Environment Sciences (ICCES) 
Institute of Atmospheric Physics, Chinese Academy of Sciences 
Office 406, Building 3, No 81 Beichen West Road, Chaoyang District, Beijing 100029, China 
Phone: +86-10-82995282 
```

**Reply by mdeushiMRI** [reply]
Our MRI model also needs the population density forcing dataset for fire modeling. So I would be grateful if you would provide the historical (and scenarios) population density dataset for CMIP7 Fast Track experiments. I think one good idea would be to have Fang Li share the HYDE3.4 dataset with us.

**Comment by jcmt** [comment]
I'm also interested in this. I've been using HYDE (historical) and IMAGE (SSPs) within UKESM1+INFERNO as part of the development towards UKESM2.

**Comment by eleanororourke** [comment]
Latest update (29th May) on population dataset development:

**Development of new population dataset for CMIP7**

As ESMs are moving towards modelling fires interactively, thereby simulating the feedbacks on both terrestrial vegetation, carbon cycle as well as atmospheric composition. This presents some new forcing data requirements including population density data required for fire ignitions (see details in [this presentation](https://wcrp-cmip.org/wp-content/uploads/2024/10/S5_JMulcahy.pdf)).

In response to this need, a dataset will be created by collaboration of the [COMPASS](https://compass-climate.eu/) and [WorldPop](https://www.worldpop.org/) projects. It is based on the approach of merging exposure dataset described in [this document](https://compass-climate.eu/Public%20Deliverables/D3.1_Exposure%20datasets%20at%20multiple%20scales.pdf), but with the main change being the inclusion of the new WorldPop dataset. Gridded population at resolutions of 1 km or lower (such as 0.5 degree for ESMs) will utilize WorldPop gridded data for 2015-2100, with extrapolation back to 1975 with Global Human Settlement Layer (GHSL) and back to 1850 with History database of the Global Environment (HYDE; main source of data so far). As the gridded data comes in 5- or 10-year resolution, it will be interpolated. Then, the gridded data will be adjusted to annual timeseries at country level using a new compilation of historical data from project COMPASS for 1850-2023 (near-equal to UN World Population Prospects for 1950-2023) and then will follow the SSP database, specifically the upcoming SSP v3.2 data which will also be used in ScenarioMIP. The graph below schematically shows how the data is combined.

The change in the dataset is proposed to improve quality of both gridded and national data compared to HYDE and increase available resolution for impact data. Validation for Europe has shown that WorldPop and GHSL reduce the error in population distribution (measured by average error per grid cell) by more than half. At a 0.5 degree resolution, the average difference in grid-cell population compared to HYDE would be about 20%. The importance of population data quality and resolution is much larger for impact models, therefore a switch to a new data source already for ESMs is beneficial. The change will not introduce a major discrepancy in land use as it has limited relation to population data after 1960 in HYDE, while pre-1960 population trends are largely preserved (with limited exceptions to improve data accuracy).

We are currently awaiting final version of SSP v3.2 and WorldPop gridded population projections. The historical period will be provided first relatively soon, and SSP-based projections by end of June 2025.

**Comment by eleanororourke** [comment]
Relevant CMIP2026 Community Workshop session ["
Gridded Population Data and CMIP: Challenges and Opportunities"](https://airtable.com/appGEHtbRi0wy6T0x/shrS44QlaCiyLaCqA/tblZ9kfmtJ6HNMilE/viwmH8qQ2XaoBL4J3/recIhXxuZH0ORoMOe?viewControls=on) may be of interest. Call for abstracts is open at [https://wcrp-cmip.org/cmip26-call-for-abstracts/](https://wcrp-cmip.org/cmip26-call-for-abstracts/) until 13th August 2025.

**Comment by eleanororourke** [comment]
Population dataset from COMPASS and WorldPop teams for the historical (1850-2025) in two resolutions (0.25 deg, 0.5 deg) are now available [here](https://aims2.llnl.gov/search?project=input4MIPs&activeFacets=%7B%22mip_era%22:%22CMIP7%22,%22institution_id%22:%22PIK%22%7D). The future scenarios will be added as soon as emissions scenarios are officially confirmed.

Feedback from modelling centres is very much welcomed by the COMPASS (@dompap) and WorldPop (@laurencehawker) - @jcmt , @mdeushiMRI

**Comment by vnaik60** [comment]
Hi @dompap, I just noticed the metadata for pop_dens and had a minor nit. This does not affect the data in any way so an update is not necessarily needed but perhaps something to address in the files for future scenarios. 

float pop_dens(time, lat, lon) ;
		pop_dens:long_name = "population_density_per_square_km" ;
		pop_dens:units = "1" ;
		pop_dens:description = "Estimated population (mid-year) per square kilometer of grid cell area. The value was calculated 
by dividing population by the whole grid cell area, i.e. both land and water parts of the grid cell."

I think the long_name should be just "population density" and units should be "people/km2" to be precise. 
Thanks!

**Reply by dompap** [reply]
Hi @vnaik60 thank you for touching this aspect. I wasn't sure myself what to do here - my first instict was to do as you suggest. However, I had the impression that units should be written in a more standard (math-like) way, so e.g. like '1 / 1e6 m^2', given there is no standard symbol for people. But maybe indeed 'person / km^2' would be better.

**Reply by vnaik60** [reply]
Thanks Dominik! Perhaps @durack1 could provide insights based on CF-conventions on units.

**Reply by durack1** [reply]
@dompap thanks for being so diligent with this. The CF standard name table ([here](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html)) has no entries for "population", so to make this CF compliant we'd have to go through the process of requesting a CF standard name (if you want to, which I would recommend, start [here](https://github.com/cf-convention/discuss/issues)).

It might be best if we start to collect nits, like you've noted above, in addition to keeping ears open as the modelling groups start to pull down these data and start using them, we can then batch up all tweaks as required, mint a PIK-CMIP-1-0-1 (or a v1.1.0 if there are more aggregious changes, or a v2.0.0 if you're feeling versiony) and get this published, along with the format tweaks in place for the scenario datasets that will be expected down the line.

**Reply by durack1** [reply]
> Hi @vnaik60 thank you for touching this aspect. I wasn't sure myself what to do here - my first instict was to do as you suggest. However, I had the impression that units should be written in a more standard (math-like) way, so e.g. like '1 / 1e6 m^2', given there is no standard symbol for people. But maybe indeed 'person / km^2' would be better.

Just to be a pest, what about subterranean people :)

**Reply by dompap** [reply]
> @dompap thanks for being so diligent with this. The CF standard name table ([here](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html)) has no entries for "population", so to make this CF compliant we'd have to go through the process of requesting a CF standard name (if you want to, which I would recommend, start [here](https://github.com/cf-convention/discuss/issues)).
> 
> It might be best if we start to collect nits, like you've noted above, in addition to keeping ears open as the modelling groups start to pull down these data and start using them, we can then batch up all tweaks as required, mint a PIK-CMIP-1-0-1 (or a v1.1.0 if there are more aggregious changes, or a v2.0.0 if you're feeling versiony) and get this published, along with the format tweaks in place for the scenario datasets that will be expected down the line.

Alright I started the CF discussion here: https://github.com/orgs/cf-convention/discussions/425.
For sure we can update the first version of the files together with the scenario files.

**Reply by dompap** [reply]
> > Hi @vnaik60 thank you for touching this aspect. I wasn't sure myself what to do here - my first instict was to do as you suggest. However, I had the impression that units should be written in a more standard (math-like) way, so e.g. like '1 / 1e6 m^2', given there is no standard symbol for people. But maybe indeed 'person / km^2' would be better.
> 
> Just to be a pest, what about subterranean people :)

I will add to the documention that orcs and dwarves are not included :)
